cmd/agent/app/reporter/tcollector_reporter.go

```diff
@@ -63,8 +63,8 @@ type tcollectorReporter struct {
 }
 
 // NewTCollectorReporter creates new tcollectorReporter
-func NewTCollectorReporter(channel *tchannel.Channel, mFactory metrics.Factory, zlogger zap.Logger, clientOpts *thrift.ClientOptions) Reporter {
-	thriftClient := thrift.NewClient(channel, "tcollector", clientOpts)
+func NewTCollectorReporter(svc string, channel *tchannel.Channel, mFactory metrics.Factory, zlogger zap.Logger, clientOpts *thrift.ClientOptions) Reporter {
```

Comment:
looks like file/struct needs to be renamed. Isn't it more of a `TChannelReporter`? In the future we could even split the "reporter" part from "sender".